semantic-go/src/Language/Go/Tags.hs

```diff
@@ -0,0 +1,95 @@
+{-# LANGUAGE AllowAmbiguousTypes, DataKinds, DisambiguateRecordFields, FlexibleContexts, FlexibleInstances, MultiParamTypeClasses, NamedFieldPuns, PartialTypeSignatures, ScopedTypeVariables, TypeApplications, TypeFamilies, TypeOperators, UndecidableInstances #-}
```

Comment:
can we use the alternate style with one pragma per extension, each on their own line?

Reply:
Yes, that's a very good idea. I'll fix that for at least the new files I've added on this PR. 👍